docker/Dockerfile

```diff
@@ -1,52 +1,34 @@
-# FIXME: use "--platform=$BUILDPLATFORM" and solve openssl cross-compile issue
-FROM rust:1.67.0-alpine as builder
-
-# Install build dependencies
-RUN apk add --no-cache git openssl-dev libpq-dev musl-dev
-
-# Set the working directory to /app and copy the source code
+FROM clux/muslrust:1.67.0 as builder
```

Comment:
Reverting to the old dockerfile for now, since the new one builds correctly, but crashes.

Reply:
Better make a separate PR for this.

Reply:
K will do.